geo-data-server/src/main/resources/application.yml

```diff
@@ -2,7 +2,7 @@ server:
   port : 8087
 
 network-store-server:
-   base-uri: http://dev-master.pgs.rte-france.com/network-store-server/
+   base-uri: http://localhost:8080
```

Comment:
Isn't it better if we put different port for each microservice so users can try to launch it on localhost without having to change every port on every microservice ?

Reply:
Yes sure, this is the idea. Each of our micro-services normally comes with a different port. For network store server it is 8080, for geo data server it is 8087, etc. When we create a new micro-service we have to take care of that and be sure no other service has already taken this port.